source/digits_hits/include/GateRejectedVolumeMessenger.hh

```diff
@@ -0,0 +1,42 @@
+/*----------------------
+   Copyright (C): OpenGATE Collaboration
+
+This software is distributed under the terms
+of the GNU Lesser General  Public Licence (LGPL)
+See LICENSE.md for further details
+----------------------*/
+
+
+#ifndef GateRejectedVolumeMessenger_h
+#define GateRejectedVolumeMessenger_h 1
+
+#include "GatePulseProcessorMessenger.hh"
+
+class G4UIdirectory;
+class G4UIcmdWithoutParameter;
+class G4UIcmdWithAString;
+class G4UIcmdWithABool;
+class G4UIcmdWithAnInteger;
+class G4UIcmdWithADouble;
+class G4UIcmdWithADoubleAndUnit;
+class G4UIcmdWith3Vector;
+class G4UIcmdWith3VectorAndUnit;
```

Comment:
Do we really need to declare all these modules?

Reply:
I see you are using only ``G4UIcmdWithAString`` later in this class.

Reply:
You are right, I removed libraries which are not used